README.md

```diff
@@ -3,4 +3,31 @@ Plugin Installation Manager Tool
 
 [![Join the chat at https://gitter.im/jenkinsci/plugin-installation-manager-cli-tool](https://badges.gitter.im/jenkinsci/plugin-installation-manager-cli-tool.svg)](https://gitter.im/jenkinsci/plugin-installation-manager-cli-tool?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-The plugin manager will download plugins and their dependencies into a folder so that they can easily be imported into an instance of Jenkins. This tool with replace the install-plugins.sh script. 
+The plugin manager will download plugins and their dependencies into a folder so that they can easily be imported into an instance of Jenkins. This tool will replace the [Docker install-plugins.sh script](https://github.com/jenkinsci/docker/blob/master/install-plugins.sh).
```

Comment:
You can use active voice here since the tool is here!

Reply:
Fixed in 1344073.